### PR TITLE
Auth

### PR DESCRIPTION
**Pull request by Wolf-Den1994**
image

### CODE REVIEW

File: src/app/auth/auth.component.scss [hunk 1]
```diff
@@ -0,0 +1,59 @@
+@import '../../styles/variables';
+
+.login-tabs {
+  width: 100%;
+}
+
+.login-page {
+  padding: 50px 10px;
+  max-width: 1200px;
+  width: 100%;
```
Comment (algolj) [inline src/app/auth/auth.component.scss:10]: MB remove the widths. And already on the adaptive to think about these values image

File: src/app/auth/components/auth-button/auth-button.component.html [hunk 2]
```diff
@@ -12,9 +12,9 @@
     <svg class="login__authorization" viewBox="0 0 70 70" width="100%">
       <path d="M35,35m-23,0a23,23 0 1,1 46,0a23,23 0 1,1 -46,0" fill="transparent" id="tophalf" />
       <text>
-        <textPath xlink:href="#tophalf" startOffset="0%">Log in ·</textPath>
```
Comment (algolj) [inline src/app/auth/components/auth-button/auth-button.component.html:15]: Depending on whether the user is authorized, can change 'in' or 'out'.
Plus, the button itself needs to be set to routerLink in  on auth path.

File: src/app/auth/auth.component.scss (hunk 1 above)
Comment (Wolf-Den1994) [reply src/app/auth/auth.component.scss:10]: agree

File: src/app/auth/components/auth-button/auth-button.component.html (hunk 2 above)
Comment (Wolf-Den1994) [reply src/app/auth/components/auth-button/auth-button.component.html:15]: done

**Review by algolj — APPROVED** [review]

**Pull request merged**